build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -198,6 +198,10 @@ fun Test.configureJvmForTest() {
     }
     javaLauncher = launcher
     if (jvmVersionForTest().canCompileOrRun(9)) {
+        // Required by JdkTools and JdkJavaCompiler
+        jvmArgs(listOf("--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED"))
+        jvmArgs(listOf("--add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED"))
```

Comment:
💭 This is an interesting situation here: older wrappers don't have these arguments, so it should be either:
 - added here
 - a new nightly should be made with these parameters

I chose the first option and just added these flags here. 
These flags could be removed as soon as a nightly is available with the flags.

Reply:
Can you please create a follow-up issue for that?

Reply:
Documented in the enhancements issue